index/index-logic.js

```diff
@@ -31,10 +37,13 @@ module.exports = function index (options) {
   })
 
 
-  seneca.add( 'init:index', function(msg, done) {
-    index = Search({
-      db: Memdown
+  seneca.add('init:index', function(msg, done) {
+    Search({
+      indexes: Levelup('si', {db:Memdown, valueEncoding: 'json'})
```

Comment:
I don't like this because it is undocumented and seems a bit awkward, but it seems the only way to swap the levelup backend.
I've opened an issue on search-index to help clarify: https://github.com/fergiemcdowall/search-index/issues/309